app/models/pixelz/modification_request.rb

```diff
@@ -19,7 +19,20 @@ def self.raise_pixelz_error(res)
       return if res['AddImageResponse']['ErrorCode'] == 'NoError'
       raise Pixelz::Error.new res['AddImageResponse']['Message']
     end
+
+    def reject(comment)
+      res = Pixelz::ApiClient::Image.new(modifiable).reject(pixelz_ticket, comment)
+      raise_reject_pixelz_error(res)
+      res
+    end
+    
     private_class_method :raise_pixelz_error
 
+    private
+
+    def raise_reject_pixelz_error(res)
```

Comment:
Would it be better if we create a share method to raise error for above `self.raise_pixelz_error` function and `raise_reject_pixelz_error`. Both of them are private and differ only key `AddImageResponse` and `RejectImageResponse`.

```
def self.modify(image) 
  raise_pixelz_error(res, 'AddImageResponse')
  ...
end

def reject(comment)
  raise_pixelz_error(res, 'RejectImageResponse')
  ...
end

private
def raise_pixelz_error(res, key)
  return if res[key]['ErrorCode'] == 'NoError'
  raise Pixelz::Error.new res[key]['Message']
end
```